README.md

```diff
@@ -1,229 +1,124 @@
 
-   > **_NOTE:_**  In order to import the MSAL and MSAL common libraries during the private preview, you will need GitHub personal access token.
+### Step 6: Clone sample Android mobile application
 
-1. Open `app/src/main/res/raw/native_auth_sample_app_config.json` file.
-1. Find the placeholder:
+Clone the sample Android mobile application by following the steps outlined in [Clone sample Android mobile application](https://learn.microsoft.com/entra/external-id/customers/how-to-run-native-authentication-sample-android-app#clone-sample-android-mobile-application).
 
-    1. `Enter_the_Application_Id_Here` and replace it with the **Application (client) ID** of the app you registered earlier.
-    1. `Enter_the_Tenant_Subdomain_Here` and replace it with the Directory (tenant) subdomain. For example, if your tenant primary domain is `contoso.onmicrosoft.com`, use contoso. If you don't have your tenant name, learn how to [read your tenant details](https://learn.microsoft.com/azure/active-directory/external-identities/customers/how-to-create-customer-tenant-portal#get-the-customer-tenant-details).
+### Step 7: Configure the sample Android mobile application
 
-1. Select **File** > **Sync Project with Gradle Files**.
+Configure the sample Android mobile application by following the steps in [Configure the sample Android mobile application](https://learn.microsoft.com/entra/external-id/customers/how-to-run-native-authentication-sample-android-app#configure-the-sample-android-mobile-application).
 
-## Run and test sample Android mobile application
+### Step 8: Run and test sample Android mobile application
 
-To build and run your app, follow these steps:
+Run and test the Android sample mobile application by following the steps in [Run and test sample Android mobile application](https://learn.microsoft.com/entra/external-id/customers/how-to-run-native-authentication-sample-android-app#run-and-test-sample-android-mobile-application).
 
-1. In the toolbar, select your app from the run configurations menu.
-1. In the target device menu, select the device that you want to run your app on.
+### Step 9: Call a protected web API
 
-   If you don't have any devices configured, you need to either create an Android Virtual Device to use the Android Emulator or connect a physical device.
+Follow the steps in [Sign in users and call an API in a sample Android mobile app by using native authentication](https://learn.microsoft.com/entra/external-id/customers/sample-native-authentication-android-sample-app-call-web-api) to sign in users and call a protected API in the Android sample mobile app.
 
-1. Select **Run** button.
+## Key concepts
 
-1. The app will open on the email and one-time passcode screen.
+Open `app/src/main/res/raw/native_auth_sample_app_config.json` file and you find the following lines of code:
```

Comment:
Move comments from another PR
![image](https://github.com/Azure-Samples/ms-identity-ciam-native-auth-android-sample/assets/111055878/f40f94d3-cd52-4814-ba3a-8c42ed07a8b0)

Reply:
I've updated to reflect the changes.